server/recceiver/recast.py

```diff
@@ -87,6 +87,8 @@ def timed(self):
             self.phase = 2
             self.nonce = random.randint(0,0xffffffff)
             self.writeMsg(0x8002, _ping.pack(self.nonce))
+            _log.debug("ping nonce: " + str(self.nonce))
+            self.sess.proto.transport.resumeProducing()
```

Comment:
This isn't the right place to handle this.  If you want to defeat throttling, then you should do so in ```CollectionSession.flush```.

I'll accept defeating throttling for now as I realize that I didn't do it correctly for recsync.  It should be done as with LimitedSite https://github.com/epicsdeb/carchivetools/blob/master/carchive/util.py#L300 by not accept()ing additional connection attempts.  This will actually avoid hitting the per-process file descriptor limit.